python/foglamp/services/south/server.py

```diff
@@ -204,16 +204,18 @@ def run(self):
             _LOGGER.exception('Unable to stop the Ingest server. %s', str(ex))
             # return
 
-        # Finish all pending asyncio tasks
-        pending = asyncio.Task.all_tasks()
-        for p in pending:
-            p.cancel()
+        # Cancel all pending asyncio tasks after a timeout occurs
+        done, pending = await asyncio.wait(asyncio.Task.all_tasks(), timeout=5)
+        for task_pending in pending:
+            task_pending.cancel()
 
         # This deactivates event loop and
         # helps aiohttp microservice server instance in graceful shutdown
         _LOGGER.info('Stopping South Service Event Loop')
         loop.stop()
 
+        _LOGGER.info("Stopped plugin '{}'".format(self._name))
```

Comment:
Plugin shutdown logs should be (are) in its own module file; no need to duplicate again; please rebase and review